app/src/main/java/com/asfoundation/wallet/repository/GamesRepository.kt

```diff
@@ -1,25 +1,74 @@
 package com.asfoundation.wallet.repository
 
-import com.appcoins.wallet.core.network.backend.api.GamesApi
+import android.util.Log
+import com.appcoins.wallet.core.network.eskills.api.AppDataApi
+import com.appcoins.wallet.core.network.eskills.api.EskillsGamesApi
 import com.appcoins.wallet.ui.widgets.GameData
+import com.appcoins.wallet.ui.widgets.GameDetailsData
+import com.appcoins.wallet.ui.widgets.Screenshot
 import io.reactivex.Single
 import it.czerwinski.android.hilt.annotations.BoundTo
 import javax.inject.Inject
 
 @BoundTo(supertype = GamesRepositoryType::class)
-class GamesRepository @Inject constructor(private val gamesApi: GamesApi) :
+class GamesRepository @Inject constructor(
+  private val gamesApi: EskillsGamesApi,
+  private val appDataApi: AppDataApi
+) :
   GamesRepositoryType {
 
+  private val defaultBackground =
+    "https://image.winudf.com/v2/image1/Y29tLm5hdGhuZXR3b3JrLnVsdHJhcHJvX3NjcmVlbl8xXzE2MzE3MzE3MTlfMDQ5/screen-1.webp?fakeurl=1&type=.webp"
```

Comment:
why to use a image to create a background instead of creating a shape on xml?

Reply:
This variable was meant to be temporary and I used an image url because the images i get from the api are also urls. So I tried to be consistent on this, but I will try to do it with a shape.